.circleci/config.yml

```diff
@@ -90,81 +150,113 @@ jobs:
           username: $DOCKER_USER
           password: $DOCKER_PASS
         command: gcloud beta emulators bigtable start --host-port=localhost:8086
-    resource_class: xlarge
+    resource_class: large
     environment:
       BIGTABLE_EMULATOR_HOST: localhost:8086
     steps:
       - checkout
-      # Need to download the poetry.lock files so we can use their
-      # checksums in restore_cache.
-      - restore_cache:
-          name: Restoring Rust cache
-          key: rust-v1-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum "Cargo.lock" }}
-      - run:
-          name: Create Workspace
-          command: mkdir -p workspace
+      - restore_test_cache:
+          cache_key: rust-v1-integration-test-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum "Cargo.lock" }}
```

Comment:
What is the difference between the `rust-v1` we use here and the `rust-v2` we use later?

Reply:
This one is specific to the integration tests. The `v2` of the other one is because I am using a different image and the paths aren't the same